setup.py

```diff
@@ -44,10 +44,46 @@
         "GitPython>=3.1.40",
         "psutil",
     ],
+    extras_require={
+        "llm": [
+            "tqdm",
+            "torch>=2.0.0",
+            "transformers",
+            "accelerate",
+            "py-cpuinfo",
+            "sentencepiece",
+            "datasets",
+            "fastapi",
+            "uvicorn[standard]",
+        ],
+        "llm-oga-dml": [
+            "onnxruntime-directml==1.18.0",
+            "onnxruntime-genai-directml==0.2.0",
+            "tqdm",
+            "torch>=2.0.0",
+            "transformers",
+            "accelerate",
+            "py-cpuinfo",
+            "sentencepiece",
+            "datasets",
+            "fastapi",
+            "uvicorn[standard]",
+        ],
+        "llm-oga-npu": [
+            "transformers",
+            "torch",
+            "onnx==1.16.0",
+            "onnxruntime==1.18.0",
+            "numpy==1.26.4",
+            "uvicorn[standard]",
+        ],
+    },
     classifiers=[],
     entry_points={
         "console_scripts": [
             "turnkey=turnkeyml:turnkeycli",
+            "turnkey-llm=turnkeyml.llm:lemonadecli",
+            "lemonade=turnkeyml.llm:lemonadecli",
```

Comment:
Is the plan to keep multiple entry points for the same functionality long-term?

Reply:
Yeah. Can't let go of the classics.